.github/workflows/docker-build.yaml

```diff
@@ -558,6 +591,14 @@ jobs:
                         echo "All translations are dumped";
                         exit 0;
                     fi
+            -   name: Check all not previously used commands are passing
+                run: |
+                    docker compose exec -T php-fpm php phing -D production.confirm.action=y frontend-api-enable cron-list cron-run-all-serially cron-watch maintenance-on maintenance-off elasticsearch-export-changed
+                    docker compose exec -T php-fpm php bin/console shopsys:categories:recalculate
+                    docker compose exec -T php-fpm php bin/console shopsys:cdn-domain-url:replace
+                    docker compose exec -T php-fpm php bin/console shopsys:clean-opcache
+                    docker compose exec -T php-fpm php bin/console shopsys:database:dump dump.sql
+                    docker compose exec -T php-fpm php bin/console shopsys:domains:info
```

Comment:
is not it possible to wrap all these lines into a file or script so you do need to repeat yourself? ... now I see there is one difference - `frontend-api-enable` - this is on purpose?

Reply:
I know, that our GitHub actions workflows can be lengthy. Still, in this case, I prefer to have that code directly visible when editing such files, because click-through is not present there, and searching and opening included files feels to be annoying.